README.md

```diff
@@ -272,6 +273,12 @@ Alternatively you can pull from [Docker Hub](https://hub.docker.com/r/kuchy/pack
 docker pull kuchy/packtpub-crawler
 ```
 
+### cronjob
+Add this to your crontab to run the job daily at 9:
+```
+00 09 * * * /usr/bin/python PATH_TO_CRAWLER/script/spider.py --config PATH_TO_CRAWLER/config/prod.cfg --all --extras --notify join >> /tmp/packtpub.log 2>&1
+```
+
```

Comment:
yep, the command works fine without cron.

here the logs
```
Dec 13 20:16:49 dell crontab[13354]: (ubuntu) LIST (ubuntu)
Dec 13 20:17:01 dell CRON[13393]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Dec 13 20:18:01 dell CRON[13701]: (ubuntu) CMD (/usr/bin/python MY_PATH/packtpub-crawler/script/spider.py --config MY_PATH/packtpub-crawler/config/prod.cfg)
Dec 13 20:18:12 dell CRON[13700]: (CRON) info (No MTA installed, discarding output)
```

of course MY_PATH is the real value